src/components/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,229 @@
+import React, {
+  Children,
+  ReactElement,
+  useEffect,
+  useRef,
+  useState,
+} from 'react'
+import classnames from 'classnames'
+interface TooltipProps {
+  label: string
+  position?: 'top' | 'bottom' | 'left' | 'right'
+  className?: string
+  children: ReactElement
+}
+
+const TRIANGLE_SIZE = 5
+const SPACER = 2
+const tooltipID = `tooltip-${Math.floor(Math.random() * 900000) + 100000}`
```

Comment:
I _think_ this should probably be initialized inside the `Tooltip` component, probably inside a `useEffect` hook so it just happens once on initial mount.